### DISCUSSION TITLE
Using fxtest

### DISCUSSION THREAD

**Discussion by mehdihadeli** [uber-go/fx#1013] [Q&A]
Hi,
Are there any docs or sample for using `fxtest` in our tests?

**Comment by sywhang** [comment]
Hi @mehdihadeli, we're aware that there is currently no docs or guides on how you can use `fxtest`. I've filed https://github.com/uber-go/fx/issues/1026 to track this work, but I probably won't be able to get to it till at least February due to some other priorities that need immediate attention.

But in the meantime you can take a look at some of the unit tests that make use of `fxtest`. Specifically, you can look at tests in `app_test.go` and various tests in the `docs` directory where we keep the tests for the sample code that we present in our docs site. 

I'll update this thread again once we get to fixing this issue in our documentation.

**Reply by mehdihadeli** [reply]
Hi,
Thanks, I will take a look to `app_test.go`.
